lib/sample_uploader/sample_uploaderImpl.py

```diff
@@ -767,6 +767,36 @@ def filter_samplesets(self, ctx, params):
                              'output is not type dict as required.')
         # return the results
         return [output]
+
+    def get_sampleset_meta(self, ctx, params):
+        """
+        :param params: instance of type "GetSamplesetMetaParams" (Get list of
+           metadata keys/columns from a given sampleset. Used to populate
+           filter_sampleset dynamic dropdown with valid options from a given
+           list of samples.) -> structure: parameter "sampleset_ref" of list
+           of String
+        :returns: instance of list of String
+        """
+        # ctx is the context object
+        # return variables are: results
+        #BEGIN get_sampleset_meta
+        samples = []
+        for sample_set in self.dfu.get_objects({'object_refs': params['sample_set_ref']})['data']:
+            samples.extend(sample_set['data']['samples'])
+        sample_ids = [{'id': sample['id'], 'version':sample['version']} for sample in samples]
+        sample_search = sample_search_api(url=self.sw_url, service_ver="dev")
+        results = sample_search.get_sampleset_meta({
```

Comment:
`sample_search_api` client needs to be reinstalled so that `get_sampleset_meta` is available. Currently, only `filter_samples` is callable.

Reply:
I'm assuming I can't reinstall the client until the new method is released?

Reply:
You are right. Tests won't pass until the new method is released. We can revisit this PR once the new method is released. You can still install the client by running `kb-sdk install https://raw.githubusercontent.com/kbase/sample_search_api/master/sample_search_api.spec`

Reply:
sample_search_api is already installed for the filter_samplesets method, so I think it would make sense to wait until a release is out that has get_sampleset_meta